packages/merge/src/modules/merge/actions.js

```diff
@@ -64,9 +64,11 @@ export const executeMerge = () => ({
   type: EXECUTE_MERGE
 })
 
-export const setMergeResponse = mergeResponse => ({
+export const setMergeResponse = (notCopiedRelations, notDeletedEntities, showPermissionMessage) => ({
```

Comment:
Wouldn't it be simpler and cleaner to just pass a "mergeResponse" object like before?

This would have the advanced that only one parameter is used and no custom reducer is required.

"done" could be selected like this:

```
const mapStateToProps = (state, props) => ({
  mergeDone: !!state.merge.mergeResponse
})	})
```

But that required mergeResponse to be initialized with null.